test/configCases/html/parser-processing-instructions/page.html

```
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<body>
+	<?instruction?>
+	<?instruction data and more?>
+	<?instruction ? still data?>
+	<?instruction terminated by gt>
+	<?xml version="1.0"?>
+	<?1st-place?>
+	<?dangling
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)